Modules/System/Azure Storage Services Authorization/src/SAS/SASParameters.Table.al

```diff
@@ -50,5 +50,16 @@ table 9064 "SAS Parameters"
             OptionMembers = "https&http","https";
             OptionCaption = '"https and http",https', Locked = true;
         }
+
+        /// <summary>
+        /// Indicates the encryption scope to use to encrypt the request contents.
+        /// This field is supported with version 2020-12-06 and later.
+        /// See also: https://learn.microsoft.com/en-us/rest/api/storageservices/create-account-sas#specify-the-encryption-scope
+        /// </summary>
+        field(5; SignedEncryptionScope; Text[63])
```

Comment:
How did you land on a length of 63? Can't seem to find that anywhere...

Reply:
I did only found this reference on other azure documentations.
And I guessed that it would be everywhere identical:
https://learn.microsoft.com/en-us/dotnet/api/microsoft.azure.management.storage.iencryptionscopesoperations.getwithhttpmessagesasync?view=azure-dotnet

https://learn.microsoft.com/en-us/javascript/api/preview-docs/@azure/arm-storage/encryptionscopes?view=az-js-storage-v12

Reply:
See also: https://learn.microsoft.com/en-us/rest/api/storagerp/encryption-scopes/put?tabs=HTTP#uri-parameters

I updated the documentation of this property.